### DISCUSSION TITLE
Problem to download the json from the data_url variable

### DISCUSSION THREAD

**Discussion by isudoajl** [kyleskom/NBA-Machine-Learning-Sports-Betting#72] [General]
First of all congratulations for this great project!

Everything was fine until today, does anyone know what happened?

Problem to download the json from the data_url variable in the main.py.

**Comment by kyleskom** [comment]
Trying doing a git pull and re installing the requirements.txt

**Reply by isudoajl** [reply]
Thanks in advance for your quickly reply, Im getting this error from google collab:

```
WARNING:tensorflow:SavedModel saved prior to TF 2.5 detected when loading Keras model. Please ensure that you are saving the model with model.save() or tf.keras.models.save_model(), *NOT* tf.saved_model.save(). To confirm, there should be a file named "keras_metadata.pb" in the SavedModel directory.
An error occurred: Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "main.py", line 106, in <module>
    main()
  File "main.py", line 62, in main
    odds = SbrOddsProvider(sportsbook=args.odds).get_odds()
  File "/content/NBA-Machine-Learning-Sports-Betting/NBA-Machine-Learning-Sports-Betting/NBA-Machine-Learning-Sports-Betting/src/DataProviders/SbrOddsProvider.py", line 13, in __init__
    self.games = Scoreboard(sport="NBA").games
AttributeError: 'Scoreboard' object has no attribute 'games'
```

**Comment by kyleskom** [comment]
Will be looking into this issue.

**Reply by salive** [reply]
Same issue. It looks like it could not retrieve the data from sportbook

**Reply by sauceboyjr** [reply]
Hello Mr. Kyle, first of all thank you for this great project. Just wanted to let you know that we are having this particular AttributeError from 'Scoreboard' as well. Seems to be a problem with the **_sbrscrape_** library. Best regards.

**Comment by isudoajl** [comment]
Hello..
They already fixed it in the last commit they made.  It was a problem with
the version of the sbrscrape library in the requirements.txt file you need
to change to sbrscrape==0.0.4

**Reply by sauceboyjr** [reply]
thank you man!!!
